src/xenia/kernel/xam/xam_user.cc

```diff
@@ -752,6 +752,35 @@ dword_result_t XamUserCreateStatsEnumerator_entry(
 }
 DECLARE_XAM_EXPORT1(XamUserCreateStatsEnumerator, kUserProfiles, kSketchy);
 
+dword_result_t XamProfileFindAccount_entry(qword_t offline_xuid,
+                                           pointer_t<X_XAMACCOUNTINFO> account,
+                                           lpdword_t device_id) {
+  if (!account) {
+    return X_ERROR_INVALID_PARAMETER;
+  }
+
+  memset(account, 0, sizeof(X_XAMACCOUNTINFO));
+
+  const auto& profiles =
+      kernel_state()->xam_state()->profile_manager()->GetProfiles();
```

Comment:
To be fair. I think I need to rename that from GetProfiles to GetAccounts. Profile kinda indicates that it is already mounted and it is a bit confusing